### PR TITLE
enhancement - add groupId, artifactId and version attributes

### PR DESCRIPTION
**Pull request by mamilic**
This relates to, https://github.com/microsoft/vscode-java-dependency/issues/857

### CODE REVIEW

**Comment by mamilic** [comment]
Hi @jdneo, I was wondering where should I write the test for this change?

**Comment by jdneo** [comment]
We don't have tests for this module right now. I would be great if you would like to add some, but it won't block this PR to be merged. We can do it in another PR.

BTW, since this is a plugin of the JDT.LS, which uses OSGi for build, adding test cases might be a little bit complex.

File: extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java
Comment (jdneo) [inline extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java]: What will happen if the value part, (mavenDependencyModule.getOrganization()) is `null`?

Comment (mamilic) [reply extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java]: Hm, good question. Do we want to include attributes only if not null and not empty?

Comment (jdneo) [reply extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java]: Yes, it feels safer to only append attributes when they are not blank.

**Review by jdneo — DISMISSED** [review]

**Comment by jdneo** [comment]
Hi @mamilic,

After this PR merged, do you need me to publish a pre-release which includes this change for the purpose of Java Projects view integration?

**Review by jdneo — APPROVED** [review]

**Comment by mamilic** [comment]
Hi @jdneo , that would be awesome, then I can work on Java Projects View. Thanks!

**Comment by jdneo** [comment]
@mamilic I plan to merge this PR after 3.17.0 is released. Currently there is some regression happens at the build server side. I may need to revert some changes which may take some time.

**Comment by mamilic** [comment]
Hi @jdneo , if you have time please let me know when pre-release is available with this changes included.

**Pull request merged**
